bin/deepstate/main_angr.py

```diff
@@ -360,13 +360,15 @@ def hook_apis(project, run_state):
   return mc, apis
 
 
-def main_take_over(args, project):
-  takeover_ea = find_symbol_ea(project, 'DeepState_TakeOver')
+def main_take_over(args, project, takeover_symbol):
+  takeover_ea = find_symbol_ea(project, takeover_symbol)
 
-  hook_function(project, takeover_ea, TakeOver)
+  if not args.klee:
+      hook_function(project, takeover_ea, TakeOver)
 
   if not takeover_ea:
-    L.critical("Cannot find symbol `DeepState_TakeOver` in binary `{}`".format(
+    L.critical("Cannot find symbol `{}` in binary `{}`".format(
+        takeover_symbol,
```

Comment:
i `L` is a standard Python logging module then we don't need and we shouldn't use `.format` here at all. Just: 
```python
L.critical("Cannot find symbol '%s' in binary '%s'", takeover_symbol, args.binary)
```

The logging module will take care of formatting the string if this particular log should appear in logs (e.g. if we have proper logging level).

Reply:
Hmm, that's true. It's too bad that (since we're stuck with Python 2) we can't use the more modern `str.format()` [format style](https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles), so we could get the best of both worlds.

I suppose we could go ahead and make this change in a new PR, since what you suggest is a more idiomatic use of the `logging` module.